README.md

```diff
@@ -39,6 +40,29 @@
     - Use it if at any point in time, for whatever reason, the saved token gets rejected by Dualis.
     - It does not affect the rest of the config.
     - It prints all information into the console.
+- `python main.py --new-token --email wi@dhbw.de --password 1234`
+    - Overrides the login-token with a newly obtained one. With the specified credentials
+    - It does not fetch the current state of your Dualis-Account.
+    - Use it if at any point in time, for whatever reason, the saved token gets rejected by Dualis.
+    - It does not affect the rest of the config.
+    - It prints all information into the console.
+    - Be aware, that both your password and the email will be _*exposed*_ in the system logs and in the crontab if you use this. Only use in trusted environments.
+- `python main.py --new-token --file credentials.json`
+    - Overrides the login-token with a newly obtained one. With the specified credentials
+    - It does not fetch the current state of your Dualis-Account.
+    - Use it if at any point in time, for whatever reason, the saved token gets rejected by Dualis.
+    - It does not affect the rest of the config.
+    - It prints all information into the console.
+    - Loads creds from specified file
+    - Be aware, that both your password and the email will be _*exposed*_ in the file you are loading.
+    - specify the credentials as follows in the file:
```

Comment:
Capital S at the start of the bullet point for consistency, while we're at it ;)